Automattic/wp-calypso
# Remove/wp script dependency

- Remove @wordpress/scripts dependency which was bloating node_modules for tiny bit of functionality.
- Duplicate necessary functionality (`camelCaseDash`)
- Bump package versions.

#### Testing instructions

* The externals webpack plugin continues to work as expected. Try building `o2-blocks`:

`npx lerna run build --scope='*/o2-blocks'`
